File: LatA/ram.cpp
#include <iostream>
#include <string>
using namespace std;
class ram{
    // Attribute
    private:
        string kapasitas;
        int price;
    public:
    //Constructor kosong
    ram(){
    }
    // Constructor
    ram(string capacity, int harga){
        this->kapasitas = capacity;
        this->price = harga;
    }
    // Get dan Set
    void setKapasitas(string capacity){
        this->kapasitas = capacity;
    }
    string getKapasitas(){
        return this->kapasitas;
    }
    void setPrice(int harga){
        this->price = harga;
    }
    int getPrice(){
        return this->price;
    }
    // Fungsi untuk menampilkan isi dari tiap attribute
    void printRam(){
        cout << "RAM Capacity: " << this->kapasitas << endl;
        cout << "RAM Price: " << this->price << endl;
    }
    // Destructor
    ~ram(){}
};

File: LatB/Python/Ram.py
class Ram():
    # Constructor
    def __init__(self):
        # Attribute-attribute
        self.kapasitas = ""
        self.harga = 0
    # Get dan Set
    def setKapasitas(self, kapasitas):
        self.kapasitas = kapasitas
    def getKapasitas(self):
        return self.kapasitas
    def setHarga(self, harga):
        self.harga = harga
    def getHarga(self):
        return self.harga
    # Fungsi untuk menampilkan isi dari attribute
    def printRam(self):
        print("RAM Capacity: " + str(self.kapasitas))
        print("RAM Price: " + str(self.harga))